stepengine-api/src/main/java/org/tweetwallfx/stepengine/api/config/StepEngineSettings.java

```diff
@@ -204,4 +224,55 @@ public String toString() {
             ), super.toString());
         }
     }
+
+    /**
+     * Configurable object containing configuration data (via
+     * {@link #getConfig()} or {@link #getConfig(java.lang.Class)}) for a
+     * {@link Visualization} instance (identified via
+     * {@link #getVisualizationClassName()}.
+     *
+     * <p>
+     * Configuration can be extended by configuring the properties of the
+     * {@code config} section of this definition on the root level of the
+     * Configuration.
+     */
+    public static final class VisualizationSetting extends ConfigurableObjectBase {
+
+        private String visualizationClassName;
+
+        /**
+         * Returns the class name of the {@link Visualization}.
+         *
+         * @return the class name of the {@link Visualization}
+         */
+        public String getVisualizationClassName() {
+            return visualizationClassName;
+        }
+
+        /**
+         * Sets the class name of the {@link Visualization}
+         *
+         * @param visualizationClassName the class name of the
+         * {@link Visualization}
+         */
+        public void setVisualizationClassName(final String visualizationClassName) {
+            this.visualizationClassName = visualizationClassName;
+        }
+
+        @Override
+        public <T> T getConfig(final Class<T> typeClass) {
+            @SuppressWarnings("unchecked")
+            final Map<String, Object> specializedConfig = (Map<String, Object>) Configuration.getInstance().getConfig(typeClass.getName(), Collections.emptyMap());
+            final Map<String, Object> mergedConfig = Configuration.mergeMap(getConfig(), specializedConfig);
+            return JsonDataConverter.convertFromObject(mergedConfig, typeClass);
+        }
```

Comment:
I would not override this method. Line 265 loads data from a global configuration point which may be undesirable for a view class that can be configured as the type of two or more visualizations.

Reply:
Agreed